android-exoplayer/src/main/java/com/brentvatne/exoplayer/ReactTVExoplayerViewManager.java

```diff
@@ -524,7 +524,7 @@ public void receiveCommand(final ReactTVExoplayerView root, int commandId, @Null
                 root.seekTo(args.getString(0));
                 break;
             case COMMAND_SEEK_TO_POSITION:
-                root.seekTo(args.getInt(0));
+                root.seekTo(args.getInt(0) * 1000);
```

Comment:
need to check whether tv send ms or s to player

Reply:
It's sending seconds for some reason. Which is why I needed to multiply by 1000

Reply:
Could also change this on the JS layer, but not sure if that would have side-effects for tvOS?